### PR TITLE
cross entropy loss

### PR DESCRIPTION
**Pull request by SoluMilken**
input: word index
recon loss : cross entropy
warning: this model needs great amount of memory.

### CODE REVIEW

File: text_autoencoder/variational_autoencoders/lstm2lstm_2.py [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+        self.start_of_word_index = start_of_word_index
+        self.end_of_word_index = end_of_word_index
+
+        self.initialize_graph = initialize_graph
+        if self.initialize_graph:
+            super(VAELstm2Lstm_2, self).__init__(
+            )
+
+    def build_graph(self):
+        x_place = tf.placeholder(
```
Comment (SoluMilken) [inline text_autoencoder/variational_autoencoders/lstm2lstm_2.py]: 差別在這

File: text_autoencoder/variational_autoencoders/lstm2lstm_2.py [hunk 2]
```diff
@@ -0,0 +1,728 @@
...
+         0 < max_value < 1
+         step: tf.float32
+        """
+        return max_value / (1 + tf.exp(n_iterations - tf.cast(step, tf.float32)))
+
+    def compute_loss(self, x, z_mean, z_std, decoder_output_logits, step):
+        with tf.variable_scope("loss"):
+            # reconstruction loss: E[log P(X|z)]
+            recon_loss = tf.reduce_mean(
+                input_tensor=tf.nn.softmax_cross_entropy_with_logits(
```
Comment (SoluMilken) [inline text_autoencoder/variational_autoencoders/lstm2lstm_2.py]: 差別在這裡
